source_control/gitlab_project.py

```diff
@@ -0,0 +1,391 @@
+            - Possible values are: true and false.
+        required: false
+        default: true
+    wiki_enabled:
+        description:
+            - If an wiki for this project should be available or not.
+            - Possible values are: true and false.
+        required: false
+        default: true
+    snippets_enabled:
+        description:
+            - If creating snippets should be available or not.
+            - Possible values are: true and false.
+        required: false
+        default: true
+    public:
+        description:
+            - If the project is public available or not.
+            - Setting this to true is same as setting visibility_level to 20.
+            - Possible values are: true and false.
+        required: false
+        default: false
+    visibility_level:
+        description:
+            - Private. visibility_level is 0. Project access must be granted explicitly for each user.
+            - Internal. visibility_level is 10. The project can be cloned by any logged in user.
+            - Public. visibility_level is 20. The project can be cloned without any authentication.
+            - Possible values are: 0, 10 and 20.
+        required: false
+        default: 0
+    import_url:
+        description:
+            - Git repository which will me imported into gitlab.
+            - Gitlab server needs read access to this git repository.
+        required: false
+        default: false
+    state:
+        description:
+            - create or delete project.
+            - Possible values are: present and absent.
+        required: false
+        default: "present"
+        choices: ["present", "absent"]
+'''
+
+EXAMPLES = '''
+- name: "Delete Gitlab Project"
+  local_action: gitlab_project
+                server_url="http://gitlab.dj-wasabi.local"
+                verify_ssl=false
+                login_token="WnUzDsxjy8230-Dy_k"
+                name=my_first_project
+                state=absent
+
+- name: "Create Gitlab Project in group Ansible"
+  local_action: gitlab_project
+                server_url="https://gitlab.dj-wasabi.local"
+                verify_ssl=true
+                login_user=dj-wasabi
+                login_password="MySecretPassword"
+                name=my_first_project
+                group=ansible
+                issues_enabled=false
+                wiki_enabled=true
+                snippets_enabled=true
+                import_url="http://git.example.com/example/lab.git"
+                state=present
+'''
+
+try:
+    import gitlab
+    HAS_GITLAB_PACKAGE = True
+except:
+    HAS_GITLAB_PACKAGE = False
+
+
+class GitLabProject(object):
+    def __init__(self, module, git):
+        self._module = module
+        self._gitlab = git
+
+    def createOrUpdateProject(self, project_exists, group_name, import_url, arguments):
+        is_user = False
+        group_id = self.getGroupId(group_name)
+        if group_id == '':
```

Comment:
did we misunderstood? I just wanted to say instead of 
~~~
if group_id == '':
~~~
this can be simplified (and would also work for None values):
~~~
if not group_id:  
~~~